src/WinRT.Runtime/Projections/SystemTypes.cs

```diff
@@ -55,21 +55,26 @@ public struct DateTimeOffset
         // 'System.DateTimeOffset.Ticks' is relative to 01/01/0001
         public long UniversalTime;
 
+        // Numer of ticks counted between 0001-01-01, 00:00:00 and 1601-01-01, 00:00:00.
+        // You can get this through:  (new DateTimeOffset(1601, 1, 1, 0, 0, 1, TimeSpan.Zero)).Ticks;
+        private const long ManagedUtcTicksAtNativeZero = 504911232000000000;
```

Comment:
Can you add to this [DateTime test](https://github.com/microsoft/CsWinRT/blob/d361800d2cf6a46d2c7d276126a09bab1d8bc1c4/src/Tests/UnitTest/TestComponentCSharp_Tests.cs#L1912) to exercise a negative value where these changes help.  You can follow the same format where you set the property and then retrieve it and compare.